src/domain/usecases/data-entry/amc/utils/__tests__/calculationConsumptionSubstanceLevelData.spec.ts

```diff
@@ -34,7 +34,7 @@ describe("Given calculate Consumption Substance Level Data function", () => {
             const period = "2019";
             const orgUnitId = "vboedbUs1As";
             const rawSubstanceConsumptionData = givenRawSubstanceConsumptionDataByType(type);
-            const atcVersionsByKeys = atcVersionsByKeysData as ListGlassATCVersions;
+            const atcVersionsByKeys = atcVersionsByKeysData as unknown as ListGlassATCVersions;
```

Comment:
These two forced casting may be introducing bugs. Review, it seems PATH should be defined as optional

Reply:
Thanks for this because I've found that `ATCAlterationsData` type was not correctly typed and also in this json, ATC data was missing `PATH` (not used in these tests but it has to be in ATC data)